File: JsonInflatorAndroid/JsonInflatorExample/src/main/java/com/crescentflare/jsoninflatorexample/ViewletCreator.java
package com.crescentflare.jsoninflatorexample;

import com.crescentflare.jsoninflator.JsonInflator;

import org.jetbrains.annotations.NotNull;

/**
 * A custom json inflator for views
 */
public class ViewletCreator extends JsonInflator
{
    public static ViewletCreator instance = new ViewletCreator("viewlet", "viewletStyle");

    private ViewletCreator(@NotNull String inflatableKey, @NotNull String attributeSetKey)
    {
        super(inflatableKey, attributeSetKey);
    }
}

File: JsonInflatorIOS/Example/JsonInflator/ViewletCreator.swift
//
//  ViewletCreator.swift
//  Json inflator example
//
//  A custom json inflator for views
//

import UIKit
import JsonInflator

class ViewletCreator: JsonInflator {

    static let shared = ViewletCreator(inflatableKey: "viewlet", attributeSetKey: "viewletStyle")

}
